operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -597,7 +597,16 @@ local utils = import 'mixin-utils/utils.libsonnet';
     )
     .addPanel(
       $.panel('Inflight requests') +
-      $.queryPanel('sum(cortex_bucket_stores_gate_queries_in_flight{%s, component="%s"})' % [$.namespaceMatcher(), component], 'Total')
+      $.queryPanel(
+        [
+          'sum(cortex_bucket_stores_gate_queries_in_flight{%s, component="%s"})' % [$.namespaceMatcher(), component],
```

Comment:
This query has nothing to do with the object storage. This query measure the number of inflight queries in the store-gateway, while `getObjectStoreRows()` is used to build views over the object storage (e.g. S3).

Reply:
@replay made the argument that it should be added to the object storage method in the original PR #2914

Reply:
@pracucci Would you prefer I go with the original approach in #2914 and add the panel for store-gateway inflight requests only to the reads and queries dashboards?

Reply:
I think #2914 is wrong. The point is that `getObjectStoreRows()` is used for any object store client (e.g. ingesters, compactor, ...) while `cortex_bucket_stores_gate_queries_in_flight` is a store-gateway specific metric. The usage of `cortex_bucket_stores_gate_queries_in_flight` shouldn't not be part of `getObjectStoreRows()` cause it doesn't measure the inflight requests to the object storage (as this PR title says too) but it measure the number of inflight queries to the store-gateway (which is totally unrelated from the number of inflight requests to object storage).